modules/casper.js

```diff
@@ -1498,8 +1498,9 @@ Casper.prototype.resourceExists = function resourceExists(test) {
             break;
         case "function":
             testFn = test;
-            if (phantom.casperEngine !== "slimerjs")
-                testFn.name = "_testResourceExists_Function";
+            // if (phantom.casperEngine !== "slimerjs") {
```

Comment:
:+1: , CasperJS will be the first and only tool to support triffleJS, phantomJS (1&2) and slimerJS.